pages/_app.js

```diff
@@ -73,6 +88,21 @@ const GlobalStyles = createGlobalStyle`
   }
 `
 
+const REFACTORED_PATHS = ['/maker/apps', '/maker/apps/blockstack-only', '/submit-your-app'];
+
+const RenderRouteThemeProvider = ({ children }) => {
```

Comment:
Nice! Interesting way to gradually introduce new styles.

Reply:
It isn't perfect, still some weird behaviours, but a step in the right direction.